tests/periph_timer_periodic/main.c

```diff
@@ -61,7 +63,9 @@ static void cb(void *arg, int chan)
 {
     unsigned c = count[chan]++;
 
-    printf("[%d] tick\n", chan);
+    print("[", 1);
+    print_u32_dec(chan);
+    puts("] tick");
```

Comment:
Hm, wouldn't `print_str("] tick\n");` fit better here?

Reply:
Good suggestion. Replaced both the `print` and the `puts` with `print_str`